js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -3043,4 +3043,7 @@ chooseBindingType=Choose binding type
 selectFlowType=Select flow type
 selectClientAssertionSigningAlg=Select client assertion signing algorithm
 resourceDetailsTypeHelp=The type of this resource. It can be used to group different resource instances with the same type.
-searchForClientPolicy=Search for client policy
\ No newline at end of file
+searchForClientPolicy=Search for client policy
+selectVendor=Select vendor
+keycloakSpinner=Keycloak spinner
```

Comment:
We should probably use a text here descriptive of the activity, such as "Loading"

Reply:
Good idea! Thanks @jonkoops